test-suite-http-server-tck-function-aws-api-proxy/src/test/java/io/micronaut/http/server/tck/lambda/tests/MicronautLambdaHandlerHttpServerTestSuite.java

```diff
@@ -0,0 +1,13 @@
+package io.micronaut.http.server.tck.lambda.tests;
+
+import org.junit.platform.suite.api.ExcludeClassNamePatterns;
+import org.junit.platform.suite.api.SelectPackages;
+import org.junit.platform.suite.api.Suite;
+import org.junit.platform.suite.api.SuiteDisplayName;
+
+@Suite
+@SelectPackages("io.micronaut.http.server.tck.tests")
+@SuiteDisplayName("HTTP Server TCK for Function AWS API Proxy")
+@ExcludeClassNamePatterns(value = "io.micronaut.http.server.tck.tests.RemoteAddressTest|io.micronaut.http.server.tck.tests.cors.CorsSimpleRequestTest")
```

Comment:
These two tests. I am not sure if they are real failures. But I would like to merge this PR an investigate after.